bracketcolors/src/BracketMap.cc

```diff
@@ -0,0 +1,143 @@
+/*
+ *      BracketMap.cc
+ *
+ *      Copyright 2013 Asif Amin <asifamin@utexas.edu>
```

Comment:
2013? I guess it's a typo :).
If so, ideally change all occurrences.

Reply:
thanks for catching this, fixed!